addons/udes_stock/tests/test_stock_picking_type.py

```diff
@@ -0,0 +1,14 @@
+from . import common
+
+
+class TestStockPickingCommon(common.BaseUDES):
+    def test_writing_to_picking_type(self):
+        Pickingtype = self.env["stock.picking.type"]
+
+        goods_out = Pickingtype.search([("name", "=", "Goods Out")])
+        goods_out_prefix = goods_out.sequence_id.prefix
+        self.assertEqual(goods_out_prefix, "OUT")
+
+        goods_out.write({"name": "Changed goods out", "sequence_code": "out"})
+        goods_out_prefix = goods_out.sequence_id.prefix
+        self.assertEqual(goods_out_prefix, "OUT")
```

Comment:
`Can` we check the full name of the picking? WH/OUT/ 
because we only test the prefiexs.

Reply:
Yep, I can do this for a picking so we can see the full name. However the warehouse code won't be included in the name. WH/OUT/XXXX1 is the Odoo style of name. The UDES style name is OUTXXXX1. 

Creating multi-warehouse sequence names will be handled in this story https://taiga.unipart.digital/project/udes-x/us/1898.

Reply:
Can we use a regex then? The only thing is because, before the issue was the prefix often existed in the picking name, but not as expected.

Reply:
yep should be done now